src/main/java/com/tenx/ms/retail/order/rest/dto/Order.java

```diff
@@ -0,0 +1,56 @@
+package com.tenx.ms.retail.order.rest.dto;
+
+import com.tenx.ms.commons.validation.constraints.PhoneNumber;
+import com.tenx.ms.retail.order.enums.OrderStatus;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Data;
+import org.hibernate.validator.constraints.Email;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import java.util.Date;
+import java.util.List;
+
+@Data
+public class Order {
+    @ApiModelProperty(value = "Order id", readOnly = true)
+    private Long id;
+
+    @ApiModelProperty(value = "Store id", readOnly = true)
+    private Long storeId;
+
+    @ApiModelProperty(value = "Order date")
+    private Date orderDate;
+
+    @ApiModelProperty(value = "Order status")
+    private OrderStatus orderStatus;
+
+    @ApiModelProperty(value = "Order items")
+    @NotNull
+    private List<OrderElement> elements;
+
+    @ApiModelProperty(value = "First name of the buyer")
+    @Pattern(regexp = "^[A-Za-z]+$", message = "First name must contain only alpha characters")
+    @Valid
+    @NotNull
+    private String firstName;
```

Comment:
@jonathanmelian this one is missing min max len. Also NotEmpty is more restrictive than NotNull